lib/computer.rb

```diff
@@ -7,7 +7,9 @@ def initialize(mark, board)
   end
 
   def get_move
-    analyse_board
+    return winning_move if winning_move != nil
```

Comment:
`return winning_move if winning_move` would work too and be easier to read.

It would work because if there is a winning move, the return value of `winning_move` will be something (as opposed to `nil`). And because that something is considered to be _truthy_, the condition is satisfied.

Read: https://learn.co/lessons/truthiness-in-ruby-readme

Reply:
Great point. Thank you.